REFERENCE.md

```diff
@@ -4662,6 +4667,40 @@ The squeeze function.
 
 Returns: `Any` a new string where runs of the same character that occur in this set are replaced by a single character.
 
+### <a name="stdlibcrc32"></a>`stdlib::crc32`
+
+Type: Ruby 4.x API
+
+Run a CRC32 calculation against a given value.
+
+#### Examples
+
+##### 
+
+```puppet
+stdlib::crc32('my string') => 18fbd270
+```
+
+#### `stdlib::crc32(Variant[ScalarData, Sensitive[ScalarData]] $my_data)`
+
+Run a CRC32 calculation against a given value.
+
+Returns: `String` String
+
+##### Examples
+
+###### 
```

Comment:
I think you need to regenerate the reference

Reply:
:) should be done now.